### PR TITLE
maximum relay op V2

### PR DESCRIPTION
**Pull request by Luo-Liang**
The Relay Maximum Op

### CODE REVIEW

File: python/tvm/relay/op/tensor.py [hunk 1]
```diff
@@ -213,6 +213,24 @@ def greater_equal(lhs, rhs):
     return _make.greater_equal(lhs, rhs)
 
 
+def maximum(lhs, rhs):
+    """Right shift with numpy-style broadcasting.
```
Comment (merrymercy) [inline python/tvm/relay/op/tensor.py]: Maximum!

File: tests/python/relay/test_op_level4.py [hunk 2]
```diff
@@ -21,7 +21,7 @@ def test_cmp_type():
 
 
 def test_binary_broadcast():
-    for op in [relay.right_shift]:
+    for op in [relay.right_shift, relay.maximum]:
```
Comment (tmoreau89) [inline tests/python/relay/test_op_level4.py]: insert on a new line

File: docs/langref/relay_op.rst [hunk 3]
```diff
@@ -51,6 +51,7 @@ This level enables typical convnet models.
    tvm.relay.greater_equal
    tvm.relay.less
    tvm.relay.less_equal
+   tvm.relay.maximum
```
Comment (tmoreau89) [inline docs/langref/relay_op.rst]: add maximum to the Level 4 definitions

**Comment by Luo-Liang** [comment]
Updated.

**Comment by Luo-Liang** [comment]
Hmm any ideas what is causing build failures? The container service seems to be complaining :(

**Pull request merged**